handbook/engineering/web/index.md

```diff
@@ -88,13 +92,11 @@ After the 20th of each month, we hold a retrospective, to reflect on the past it
 At the beginning of each iteration, the engineering manager will:
 
 - Schedule the retrospective meeting
```

Comment:
Other teams have found that with 2 week cycles, you can now easily setup a recurring event.

Reply:
Yeah, I expect to set one up after our first iteration (and document it), right now I don't have a good feel yet for which schedule would work best for us for the combination of retrospective and planning meeting.